bindings/python/test/flight/system/dynamics/test_atmospheric_drag.py

```diff
@@ -125,5 +127,5 @@ def test_compute_contribution(self, dynamics: AtmosphericDrag, state: State):
 
         assert len(contribution) == 3
         assert contribution == pytest.approx(
-            [0.0, -2.53370345e-05, 3.57473816e-12], abs=5e-11
+            [0.0, -0.0000278707803890, -0.0000000000197640], abs=5e-11
```

Comment:
This is due to changing from wet mass to dry mass. I feel like the distinction is not necessary in the context of just drag tests with no thrusters.

Reply:
Also I don't think it's necessary to have tests here where the numbers match up. We can just keep is not None or == the correct size etc. something simple.

Reply:
Noticed that too. Should go away with the rebase.

> keep is not None or == the correct size etc. something simple.

I also agree with this